openshift-gitops/overlays/nerc-ocp-infra/patches/dex-config.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: Patch
+metadata:
+  name: dex-config
+spec:
+  serviceAccountRef:
+    name: patch-operator
+  patches:
+    dex-config:
+      targetObjectRef:
+        apiVersion: v1
+        kind: ConfigMap
+        name: argocd-cm
+        namespace: lars-sandbox
```

Comment:
is 'lars-sandbox' where it will actually live nerc-ocp-infra and/or production?

Reply:
It is not! That's a typo. Thanks for catching it!